packages/api/src/command/medical/organization/update-organization.ts

```diff
@@ -29,9 +33,41 @@ export const updateOrganization = async (
   const fhirOrg = toFHIR(updatedOrg);
   await upsertOrgToFHIRServer(updatedOrg.cxId, fhirOrg);
 
-  if (org.type !== "healthcare_it_vendor") {
+  if (org.type === "healthcare_provider") {
```

Comment:
OOC what motivated the change in logic? makes sense - just wondering

Reply:
A bit more defensive and our style guide encourages "positive" checks vs. negative ones to some extent. Mostly just more defensive and clear IMO.

Reply:
This is no longer relevant -- moved the check into `verifyCxProviderAccess`.